### PR TITLE
Add nilearn recipe

### PR DESCRIPTION
**Pull request by lesteve**
I followed closely what was done for nibabel in #284.

### CODE REVIEW

File: recipes/nilearn/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+source:
+  fn: nilearn-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/n/nilearn/nilearn-{{ version }}.tar.gz
+  md5: 66145792fb2aebf51e221225bd210534
+
+build:
+    number: 0
+    script: python setup.py install
+    features:
+        - vc9   # [win and py27]
```
Comment (lesteve) [inline recipes/nilearn/meta.yaml]: I took this section from the nibabel recipe. nilearn is a pure python package so do we really need this Windows compiler stuff?

Comment (ocefpaf) [inline recipes/nilearn/meta.yaml]: Nope. You can remove all that.

Comment (lesteve) [inline recipes/nilearn/meta.yaml]: Removed.

**Comment by lesteve** [comment]
This seems to be working fine AFAICS. Any further comments?

File: recipes/nilearn/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    - nibabel
+    - matplotlib
+
+test:
+  imports:
+    - nilearn
+
+about:
+  home: https://nilearn.github.io
+  license: New BSD
```
Comment (ocefpaf) [inline recipes/nilearn/meta.yaml]: I never know if the New BSD is `BSD 2-Clause` or `BSD 3-Clause`. Do you mind updating that to the right version of the licence to make it easy to recognize?

**Comment by ocefpaf** [comment]
LGTM. I am merging as is and you can perform any further change on the feedstock.

Thanks for the contribution @lesteve!

**Comment by lesteve** [comment]
Great, thanks! Out of interest, I can't see nilearn through `conda search nilearn --channel conda-forge` or in https://github.com/conda-forge/nilearn-feedstock.

Is there something else I need to do or is it just a matter of waiting for the change to propagate through the system?

**Comment by ocefpaf** [comment]
We are facing some technical issues (see  #594). As soon as master clears your recipe you should see the feedstock and the build will be uploaded to the channel.

**Comment by lesteve** [comment]
OK thanks for your quick answer, I was just curious and not in any kind of hurry.

**Comment by pelson** [comment]
Ok, finally cleared up the issue, and everything is running smoothly again. The CI should be sending you merge access to https://github.com/conda-forge/nilearn-feedstock shortly. 👍

**Comment by lesteve** [comment]
> Ok, finally cleared up the issue, and everything is running smoothly again. 

Great, thanks a lot!

> The CI should be sending you merge access to https://github.com/conda-forge/nilearn-feedstock shortly.

I don't think I have received the merge rights on nilearn-feedstock yet. Any idea how is it supposed to happen, maybe as an email to my github address?

**Comment by jakirkham** [comment]
> I don't think I have received the merge rights on nilearn-feedstock yet. Any idea how is it supposed to happen, maybe as an email to my github address?

Yes, @lesteve, an email should have arrived at whatever address you use with GitHub inviting you to join `conda-forge`. Please let us know if that is not the case. Once you accept that invitation, you will be automatically added to the `nilearn` team, which has commit privileges on the [nilearn-feedstock]( https://github.com/conda-forge/nilearn-feedstock ). In the future, there will be no action required on your part after adding your handle to a recipe to become a maintainer.

**Pull request merged**